components/styles.js

```diff
@@ -49,6 +49,7 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     padding: 5,
     width: DEVICE_WIDTH / 7,
+    height: DEVICE_WIDTH / 7,
```

Comment:
Best case would be to make this auto flex to the height of the week row. Maybe removing height and making it setting `flex: 1` on it might work?